spec/featured-spec.js

```diff
@@ -4,10 +4,6 @@ var yaml = require('yamljs')
 
 test('Limit featured homepage apps to 24', function (t) {
   t.plan(1)
-
-  var apps = yaml.load(path.join(__dirname, '..', '_data', 'apps.yml'))
-  var featuredApps = apps.filter(function (app) {
-    return app.featuredOnHomepage
-  })
+  var featuredApps = require(path.join(__dirname, '../_data/featured_apps.json'))
```

Comment:
Hmm.  Looks like I was actually wrong...seems like I remembered this not working, but I just tested and path.join fixes the slashes now.  Sorry!